### PR TITLE
Pqm lcd

### PR DESCRIPTION
**Pull request by rbudai98**
## Pull Request Description

Please replace this with a detailed description and motivation of the changes. 
You can tick the checkboxes below with an 'x' between square brackets or just check them after publishing the PR. 
If this PR contains a breaking change, list dependent PRs and try to push all related PRs at the same time.

## PR Type
- [x] Bug fix (change that fixes an issue)
- [x] New feature (change that adds new functionality)
- [ ] Breaking change (has dependencies in other repos or will cause CI to fail)

## PR Checklist
- [x] I have followed the [Coding style guidelines](http://analogdevicesinc.github.io/no-OS/drivers_guide.html#coding-style)
- [x] I have performed a self-review of the changes
- [x] I have commented my code, at least hard-to-understand parts
- [x] I have build all projects affected by the changes in this PR
- [x] I have tested in hardware affected projects, at the relevant boards
- [x] I have signed off all commits from this PR
- [ ] I have updated the documentation (wiki pages, ReadMe etc), if applies

### CODE REVIEW

File: projects/eval-pqmon/src/common/pqlib_example.c [hunk 1]
```diff
@@ -43,7 +43,7 @@
 /******************************************************************************/
 
 PQLIB_EXAMPLE pqlibExample;
-extern volatile uint8_t newSyncTimeAvailable;
+extern volatile uint8_t newSyncTimeAvailable = 0;
```
Comment (buha) [inline projects/eval-pqmon/src/common/pqlib_example.c:46]: this looks like a separate bugfix, please split it into a separate commit and document what was fixed

File: projects/eval-pqmon/src/common/pqlib_afe.c [hunk 2]
```diff
@@ -54,7 +54,6 @@ int get_afe_input()
 
 	status = afe_read_status0((uint32_t *)&pOneCycle->STATUS0);
 	if ((status == 0) && (pOneCycle->STATUS0 & BITM_STATUS0_RMSONERDY)) {
-		pOneCycle->timestamp = get_irq0_timestamp();
```
Comment (buha) [inline projects/eval-pqmon/src/common/pqlib_afe.c:57]: same here, looks like it could go into a separate commit with an explanation

File: projects/eval-pqmon/src/common/afe_config.h [hunk 3]
```diff
@@ -45,6 +45,7 @@
 #include "status.h"
 #include <stdint.h>
 #include <string.h>
+#include "nhd_c12832a1z.h"
```
Comment (buha) [inline projects/eval-pqmon/src/common/afe_config.h:48]: you're adding this include but i don't see anything from it being used in this commit (you do use it in the 2nd commit, so i guess it should go there)

**Review by buha — APPROVED** [review]

**Review by amiclaus — APPROVED** [review]
